src/components/common/YourCompany/index.ts

```diff
@@ -6,5 +6,4 @@ export { default as AssociationType } from './AssociationType.vue'
 export { default as FolioInformation } from './FolioInformation.vue'
 export { default as NatureOfBusiness } from './NatureOfBusiness.vue'
 export { default as OfficeAddresses } from './OfficeAddresses.vue'
-export { default as SummaryDefineCompany } from './SummaryDefineCompany.vue'
```

Comment:
This component wasn't used anywhere. It looks like it came from Create UI originally.